src/components/ErrorPage.tsx

```diff
@@ -1,59 +1,75 @@
   "What is it about? (if a bug report please consider including your account address) %0A PLEASE FILL %0A%0A What is the feedback? %0A PLEASE FILL";
 
-// TODO: prepare for Beta release properly
 export const ErrorPage: React.FC = () => {
+  const { modalElement: OffboardingModal, onOpen: onOpenOffboardingModal } = useOffboardingModal();
+
   const onRefresh = () => {
     window.location.reload();
   };
 
-  const onBackup = () => {
-    const jsonString = `data:text/json;chatset=utf-8,${encodeURIComponent(
-      JSON.stringify(window.localStorage)
-    )}`;
+  return (
+    <Center height="100vh" padding="60px" backgroundImage={BackgroundImage} backgroundSize="cover">
+      <Box
+        width="100%"
+        maxWidth="460px"
+        height="100%"
+        maxHeight="480px"
+        padding="40px"
+        background={colors.gray[900]}
+        border="1px"
+        borderColor={colors.gray[700]}
+        borderRadius="8px"
+        boxShadow="0px 0px 30px rgba(0, 0, 0, 0.30)"
+      >
+        <ModalContentWrapper
+          icon={<NoticeIcon />}
+          subtitle="Please refresh the app or use one of the following options:"
+          title="Oops! Something went wrong!"
+        >
+          <VStack width="100%" spacing="16px">
+            <Button width="100%" onClick={downloadBackupFile} size="lg">
+              Save Backup
+            </Button>
 
-    const link = document.createElement("a");
-    link.href = jsonString;
-    link.download = "data.json";
+            <Button width="100%" onClick={onOpenOffboardingModal} size="lg" variant="warning">
+              Off-board Wallet
+            </Button>
 
-    link.click();
-  };
+            <Button width="100%" size="lg" variant="tertiary">
+              <Link
+                href={`mailto:umami-support@trili.tech?subject=Umami V2 feedback&body=${feedbackEmailBodyTemplate}`}
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                Report Error
+              </Link>
+            </Button>
 
-  const onOffboard = () => {
-    window.localStorage.clear();
-    onRefresh();
-  };
-
-  return (
-    <Flex alignItems="stretch">
-      <Center width="100%" marginTop="200px">
-        <Box>
-          <Heading marginBottom={3} textAlign="center">
-            Ooops, something went wrong!
-          </Heading>
-
-          <Button marginRight={2} onClick={onRefresh}>
-            Refresh the page
-          </Button>
-          <Button marginRight={2} onClick={onBackup} variant="secondary">
-            Download Backup
-          </Button>
-          <Button marginRight={2} onClick={onOffboard} variant="warning">
-            Offboard
-          </Button>
-          <Button variant="tertiary">
-            <Link
-              href={`mailto:umami-support@trili.tech?subject=Umami V2 feedback&body=${feedbackEmailBodyTemplate}`}
-              rel="noopener noreferrer"
-              target="_blank"
+            <Button
```

Comment:
I am not sure if this one is a button. Please confirm with Bahador. As I understand it should be something like [that](https://github.com/trilitech/umami-v2/blob/main/src/components/Onboarding/derivationPath/DerivationPath.tsx#L79-L102)

Reply:
please also check the spacing. it is too far away from the button above

Reply:
Messaged Bahador regarding this

Reply:
According to Bahador:
> The Refresh should be a button. Please follow the same spacing above and below the Refresh button as the design.

Reply:
Actually, just discussed this offline - it should be a link, the color should be changed on hover same way as for "Save Backup" button

Reply:
it's still a bit further away than the other buttons. I don't think you need this top margin
<img width="611" alt="Screenshot 2024-03-13 at 10 22 14" src="https://github.com/trilitech/umami-v2/assets/129749432/9c422fe4-71e2-42f3-8d06-bac8c09f7a79">

Reply:
Let me check this with Bahador one more time just in case